### ISSUE TITLE
Тонкости флагов

### ISSUE THREAD

**Issue by MiniGerkules** [kogutenko/common-repo#16]
Не совсем понятно, как Git разбирает, что -a -m == -am. В описании команд для git commit я не нашел отдельного флага -am. То есть это какая-то конкатенация флагов. Интересно было бы про нее узнать.

**Comment by kogutenko** [comment]
Да, ты совершенно прав. Такое работает и с другими Linux- и UNIX-программами.

В данном случае `-a` не требует аргументов, поэтому мы можем сразу записать следующий флаг `m`, минуя пробел и дефис. Лучше, наверное, показать на примере работы команды [`tar`](https://ru.wikipedia.org/wiki/Tar):

```
tar -xvf file.tar
tar -vxf file.tar
```

Здесь сочетается несколько флагов:
 - `x` говорит, что мы желаем распаковать файл (eXtract)
 - `v` даёт знать, что мы хотим видеть подробный вывод (Verbose)
 - `f` сообщает, что следом будет идти имя файла (File)

Вообще по Линуксу есть прекрасная книга Брайана Уорда «Внутреннее устройство Linux», есть перевод на русский язык.
